### DISCUSSION TITLE
Is there PVC or persistent volume claim monitoring?

### DISCUSSION THREAD

**Discussion by kisanity** [signalfx/splunk-otel-collector#2097] [Q&A] [answered]
Hello,

Is it possible to monitor container storage and PVC usage and alert based on a threshold?

We recently noticed that a postgres pod failed as it reached a max storage limit assigned to it.

Thanks

**Comment by atoulme** [comment, marked as answer]
You can collect information about container storage and PVC usage using this approach: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver#collecting-additional-volume-metadata

You can create alerts in Splunk Infrastructure Monitoring.
